grails-app/controllers/org/pih/warehouse/api/ProductApiController.groovy

```diff
@@ -274,4 +275,13 @@ class ProductApiController extends BaseDomainApiController {
         render([monthlyDemand: demand.monthlyDemand, quantityOnHand: quantityOnHand] as JSON)
     }
 
+    def getInventoryItem = {
+        InventoryItem inventoryItem = InventoryItem.findByLotNumberAndProduct(params.lotNumber, Product.findByProductCode(params.productCode))
```

Comment:
Nitpick: Change the order of the parameters.
```
findByProductAndLotNumber(Product.findByProductCode(params.productCode), params.lotNumber)
```

Reply:
With that said, sending the product ID in the API request would probably be better so we don't have to look up the product and then look up the inventory item.
```
InventoryItem.findByLotNumberAndProduct(Product.load(params.productId), params.lotNumber)
```